docs/trex_release-notes.md

```diff
@@ -11,6 +11,27 @@ layout: docs
 ----
 See also: [Known Issues]({{site.baseurl}}/docs/trex_known_issues.html)
 
+
+### Tableau 2022.3
+
+*October 2022*
```

Comment:
I would put November or December here since it didn't come out in October.

Reply:
Good point. I was slow to get this ready.

Reply:
Changed month to December